includes/class-edd-fees.php

```diff
@@ -97,7 +97,10 @@ public function add_fee( $args = array() ) {
 		$args['amount'] = edd_sanitize_amount( $args['amount'] );
 
 		// Set the fee
-		$fees[ $key ] = $args;
+		$fees[ $key ] = apply_filters( 'edd_fees_add_fees', $args, $this );
```

Comment:
[Done already, this morning](https://github.com/easydigitaldownloads/easy-digital-downloads/pull/4490/commits/d0d83153fa9e45820857e95a67d5f547908e0aed). :+1: